test/helpers/trim-off-newlines.js

```diff
@@ -0,0 +1,21 @@
+import test from '@ava/test';
+
+import trimOffNewlines from '../../lib/trim-off-newlines.js';
```

Comment:
```suggestion
import test from '@ava/test';
import trimOffNewlines from '../../lib/trim-off-newlines.js';
```

Reply:
I've update the PR with your suggestion 👍